### ISSUE TITLE
Middleman 4 support

### ISSUE THREAD

**Issue by jhass** [mscherer/middleman-piwik#2]
Title says it all, the gem currently locks on `~> 3.2`.

**Comment by mscherer** [comment]
Indeed. My co workers are looking at middleman 4 for the rest of our sites, but I have no idea if that requires code changes.

**Comment by TheE** [comment]
Any news?

**Comment by mscherer** [comment]
We decided to not go on middleman 4 for $work, so I can't justify spending too much time porting this extension, and no idea where to start (I guess just  testing should be a first good step). If someone can test and tell me "that's ok, just remove the lock", I will happily do it. 

Sorry for not communicating earlier, I kinda forgot about that extension :/
